<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>948</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>948</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1023</width>
    <height>1250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dxf2gcode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 98, 50);
color: white;
font: 15px &quot;Cascadia Code&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>500</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="Cargar_Archivo">
           <property name="styleSheet">
            <string notr="true">color:white;
background-color: rgb(55, 79, 29);
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string>Cargar Archivo DXF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5">
            <item>
             <layout class="QVBoxLayout" name="dxf_preview"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7"/>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>275</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>275</height>
         </size>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>35</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Largo :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="Alto">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>95</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color:black;</string>
         </property>
         <property name="text">
          <string>500</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Dimensiones del Bloque</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="largo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>54</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color:black;</string>
         </property>
         <property name="text">
          <string>500</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>54</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>76</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Alto :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>95</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>117</y>
           <width>301</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Configuraciones de procesamiento </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:12px;</string>
         </property>
         <property name="text">
          <string>Separación vertical entre cada moldura: </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="sep_vert">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>155</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color:black;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>136</y>
           <width>448</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:12px;
</string>
         </property>
         <property name="text">
          <string>Separación horizontal entre cada moldura: </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="sep_hor">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>202</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color:black;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>202</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>155</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Contra_moldura">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>236</y>
           <width>261</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Habilitar contra moldura</string>
         </property>
        </widget>
        <zorder>label_11</zorder>
        <zorder>label_2</zorder>
        <zorder>Alto</zorder>
        <zorder>label</zorder>
        <zorder>largo</zorder>
        <zorder>label_5</zorder>
        <zorder>label_3</zorder>
        <zorder>label_4</zorder>
        <zorder>label_8</zorder>
        <zorder>label_7</zorder>
        <zorder>sep_hor</zorder>
        <zorder>label_10</zorder>
        <zorder>label_9</zorder>
        <zorder>Contra_moldura</zorder>
        <zorder>label_6</zorder>
        <zorder>sep_vert</zorder>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>700</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="Convertir_Gcode">
           <property name="styleSheet">
            <string notr="true">color:white;
background-color: rgb(55, 79, 29);
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string>Convertir a Gcode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="styleSheet">
            <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="5">
            <item>
             <layout class="QVBoxLayout" name="dxf2gcode_vizualizer"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
